test/integration/api/RebalancingAPI.spec.ts

```diff
@@ -1058,12 +1058,36 @@ describe('RebalancingAPI', () => {
           DEFAULT_CONSTANT_AUCTION_PRICE
         );
 
+        const expectedTokenAddresses = await rebalancingSetToken.getCombinedTokenArray.callAsync();
+
+        const expectedInflowAddressArray = expectedTokenFlowArrays['inflow'].reduce((acc, unit, index) => {
+          const bigNumberUnit = new BigNumber(unit);
+          if (bigNumberUnit.gt(0)) {
+            acc.push({
+              address: expectedTokenAddresses[index],
+              unit,
+            });
+          }
+          return acc;
+        }, []);
+
+        const expectedOutflowAddressArray = expectedTokenFlowArrays['outflow'].reduce((acc, unit, index) => {
+          const bigNumberUnit = new BigNumber(unit);
+          if (bigNumberUnit.gt(0)) {
+            acc.push({
+              address: expectedTokenAddresses[index],
+              unit,
+            });
+          }
+          return acc;
+        }, []);
+
         const returnedInflowArray = JSON.stringify(returnedTokenFlowArrays['inflow']);
-        const expectedInflowArray = JSON.stringify(expectedTokenFlowArrays['inflow']);
+        const expectedInflowArray = JSON.stringify(expectedInflowAddressArray);
         expect(returnedInflowArray).to.eql(expectedInflowArray);
 
         const returnedOutflowArray = JSON.stringify(returnedTokenFlowArrays['outflow']);
-        const expectedOutflowArray = JSON.stringify(expectedTokenFlowArrays['outflow']);
+        const expectedOutflowArray = JSON.stringify(expectedOutflowAddressArray);
```

Comment:
Add a separate test case where one of the components has zero units and verify that it is excluded